test_scripts/WebEngine/WebEngineProjection/011_Verify_PTS_with_appHMIType_WEB_VIEW.lua

```diff
@@ -0,0 +1,57 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description:
+-- Check that PTS is created with appHmiType WEB_VIEW
+--
+-- Precondition:
+-- 1. SDL and HMI are started
+-- 2. WebEngine App1 with WEB_VIEW HMI type is registered
```

Comment:
Add precondition step about update of preloaded json. That application has record in preloaded table and has appropriate HMI type.

Reply:
Done in [a6a6ab7](https://github.com/LuxoftSDL/sdl_atf_test_scripts/pull/36/commits/a6a6ab72c20c10ba5a85c2ee566a1b2db7c55b87)